controller/rancher/rancher.go

```diff
@@ -158,6 +183,8 @@ func (lbc *LoadBalancerController) Run(provider provider.LBProvider) {
 
 	go lbc.LBProvider.Run(nil)
 
+	go lbc.EventsHandler.Subscribe()
```

Comment:
if event handler were to error out (for example bc the websocket connection were to be severed), we'd never restart it.

In other services, like gms, with this same paradigm, we fix by sending errors over a channel and then just letting the entire program error out. See here: https://github.com/rancher/go-machine-service/blob/master/main.go#L53-L66

BUT I don't think we want to whole container to exit because that would mean downtime for the LB. So, you could just wrap the call to Subscribe in a for loop and log the error and restart like:
```
go func() {
  for {
       err := Subscriber()
       // log error
  }
}()

Reply:
yup, makes sense.